### PR TITLE
New article: JavaScript Front-End Build Tools

### PR DESCRIPTION
**Pull request by Kristynaslomkova**

### CODE REVIEW

File: practices/team_roles_definition.md [hunk 1]
```diff
@@ -104,6 +104,7 @@ Belbin recognized that the team success depends on the behavior of the team memb
 
 - Belbin: [The Coming Shape Of Organization](http://www.belbin.ie/the-coming-shape-of-organization/)
 - Medium: [Software Project Team Roles and Responsibilities](https://medium.com/@SherrieRose/software-project-team-roles-and-responsibilities-152a7d575759)
+- Forbytes: [Making sense of the roles in a software project](https://forbytes.com/frequently-asked-questions/roles-and-responsibilities/)
```
Comment (prokopsimek) [inline practices/team_roles_definition.md]: the link is invalid - https://forbytes.com/frequently-asked-questions/roles-and-responsibilities/

remove it

File: practices/javascript-front-end-build-tools.md [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+- linting
+- dockerizing
+- packing applications
+  keywords:
+- build
+- build tools
+- front-end
+- frontend
+- front end
+  covered_by_dx_scanner: true
```
Comment (prokopsimek) [inline practices/javascript-front-end-build-tools.md]:
```suggestion
covered_by_dx_scanner: true
```

File: practices/javascript-front-end-build-tools.md [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+  short_description: If you are a front-end developer, build tools can save you valuable time. They can do almost anything: download libraries, run tests, bundle packages, automatize tasks, and so much more.
+  tags:
+- front-end build tools
+- package management
+- library
+- install
+- linting
+- dockerizing
+- packing applications
+  keywords:
```
Comment (prokopsimek) [inline practices/javascript-front-end-build-tools.md]:
```suggestion
keywords:
```

File: practices/javascript-front-end-build-tools.md [hunk 4]
```diff
@@ -0,0 +1,92 @@
+---
+slug: javascript-front-end-build-tools
+stages:
+
+- development
+  short_description: If you are a front-end developer, build tools can save you valuable time. They can do almost anything: download libraries, run tests, bundle packages, automatize tasks, and so much more.
+  tags:
```
Comment (prokopsimek) [inline practices/javascript-front-end-build-tools.md]:
```suggestion
tags:
```

File: practices/javascript-front-end-build-tools.md [hunk 5]
```diff
@@ -0,0 +1,92 @@
+---
+slug: javascript-front-end-build-tools
+stages:
+
+- development
+  short_description: If you are a front-end developer, build tools can save you valuable time. They can do almost anything: download libraries, run tests, bundle packages, automatize tasks, and so much more.
```
Comment (prokopsimek) [inline practices/javascript-front-end-build-tools.md]:
```suggestion
short_description: If you are a front-end developer, build tools can save you valuable time. They can do almost anything: download libraries, run tests, bundle packages, automatize tasks, and so much more.
```

File: practices/javascript-front-end-build-tools.md [hunk 6]
```diff
@@ -0,0 +1,92 @@
+---
+slug: javascript-front-end-build-tools
+stages:
+
```
Comment (prokopsimek) [inline practices/javascript-front-end-build-tools.md]:
```suggestion
```

File: practices/team_roles_definition.md (hunk 1 above)
Comment (Kristynaslomkova) [reply practices/team_roles_definition.md]: I tried to remove it twice in the past, I hope it's done this time!

**Comment by prokopsimek** [comment]
@Kristynaslomkova ready to merge?

**Comment by Kristynaslomkova** [comment]
@prokopsimek @DStatPersonal can we merge it? I don't know, Dan posted "Reviewed" but there's still the yellow dot next to his name in the Reviewers section (top right) that says "Awaited requested review". What does it mean? Should Dan do something?

**Pull request merged**